docs/generated/packages/gradle/documents/overview.md

```diff
@@ -14,21 +14,50 @@ Nx adds the following features to your workspace:
 - [Run only tasks affected by a PR](/ci/features/affected)
 - [Interactively explore your workspace](/features/explore-graph)
 
+{% callout type="info" title="Java Compatibility" %}
+This plugin has a minimum compatible of Java 17.
```

Comment:
I'd recommend leading with opening a GH issue as those will be viewed by the team where socials is easy to miss.